frontend/cypress/perf/common.ts

```diff
@@ -33,7 +39,12 @@ const measureLoadTime = (name: string, baseline: number, loadUrl: string, loadEl
           cy.on('uncaught:exception', (err, runnable) => {
             return false;
           });
-          cy.get(loadElementToCheck).should('be.visible');
+          if (isGraph) {
+            cy.waitForReact();
+            cy.getReact('GraphPF').should('have.length', '1');
```

Comment:
I think we should double check here that the Graph is really loaded (maybe the component `GraphPF` exists when the graph is being loaded). I would add an extra assertion here, e.g., the graph nodes length is more than 0 or the message `Graph Loading` is not displayed. Look at https://github.com/kiali/kiali/pull/7799 for some examples.

Reply:
have changed to `cy.getReact('GraphPagePFComponent', { state: { isReady: true } }).should('have.length', '1');`, the double check with `graphRefs` causes Jenkins job to fail with CPU/Memory issues.
for now having this solution with TODO